cfpb/consumerfinance.gov
# New 'products' filter for Enforcement Actions Filterable page

Allow filtering by product on the enforcement actions filter page.

## Additions

- Add "Filter enforcement products" to the Wagtail admin for filterable lists


## How to test this PR

1. Run this branch.
2. Edit the [enforcement actions filter page](http://localhost:8000/admin/pages/1327/edit/) in Wagtail.
3. For "Topic filtering", choose "Don't filter topics". For "Filter Enforcement Products", ensure the box is checked.
4. Publish the page.
5. [Visit the page](http://localhost:8000/enforcement/actions/).
6. Use the filterable list. Check that it filters by product as expected.
7. In the Wagtail admin, find the [`ELASTICSEARCH_FILTERABLE_LISTS` flag](http://localhost:8000/admin/flags/ELASTICSEARCH_FILTERABLE_LISTS/) and toggle it.
8. Back on the [filterable page](http://localhost:8000/enforcement/actions/), make sure filtering still works as expected.


## Screenshots

![Screen Shot 2021-03-15 at 1 36 42 PM](https://user-images.githubusercontent.com/4295388/111196314-86039300-8593-11eb-8589-57cbd85e2e49.png)

![Screen Shot 2021-03-15 at 1 36 14 PM](https://user-images.githubusercontent.com/4295388/111196333-8a2fb080-8593-11eb-9285-ce2ef951b7e9.png)


## Notes and todos

-


## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
